<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyWindow</class>
 <widget class="QMainWindow" name="EasyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>简单模式</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Pictures/icon.png</normaloff>../../../../Pictures/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加载文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>306</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="renishaw_button">
       <property name="text">
        <string>Renishaw</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">select_BG</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ez_button">
       <property name="text">
        <string>Ez Raman</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">select_BG</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iraman_button">
       <property name="text">
        <string>i Raman Plus</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">select_BG</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="select_BG"/>
 </buttongroups>
</ui>
